src/main/java/pages/CataloguePage.java

```diff
@@ -1,46 +1,53 @@
 package pages;
 
-import org.openqa.selenium.By;
 import org.openqa.selenium.NoSuchElementException;
 import org.openqa.selenium.WebDriver;
 import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
 
+import java.util.Arrays;
 import java.util.List;
 import java.util.Random;
 
 import static util.elementUtils.UtilActionsWithElements.clickTheElementsTimes;
 
-public class CataloguePage {
+public class CataloguePage extends BasePage {
 
     private WebDriver driver;
 
     private Random r;
```

Comment:
Please, rename this variable to smth more understandable or even remove it. Because looks like it's never used.